htdocs/comm/propal/card.php

```diff
@@ -1766,6 +1766,7 @@
 	$fk_account = GETPOST('fk_account', 'int');
 
 	// Load objectsrc
+	$soc = null;
```

Comment:
$soc is already defined line 1757.
An later the added test 
if (!empty($soc)) {
would become useless.

Reply:
Thanks, corrected it.